src/licensedcode/detection.py

```diff
@@ -978,9 +1019,13 @@ def is_false_positive(license_matches, package_license=False):
         match_rule_length == 1
         for match_rule_length in match_rule_length_values
     )
-
-    is_gpl_bare = all(
-        'gpl_bare' in license_match.rule.identifier
+    bare_rules = ['gpl_bare', 'freeware_bare', 'public-domain_bare']
+    is_bare_rule = all(
+        any([
+            True
```

Comment:
If you use this, prefer this code .... but IMHO we should have a flag on the Rule model instead
```suggestion
            bare_rule in license_match.rule.identifier
```

Reply:
Yeah this could be a computed flag in the Rule object allright, will implement this seperately. Thanks!
For now adding the suggested change.